haiku-apps/projectconceptor/projectconceptor-0.1.1.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="A basic graph application"
+DESCRIPTION="ProjectConceptor is a modular application\
+designed to view, edit and process information which\
```

Comment:
Keep one space before the backslash, otherwise it would show up as "...applicationdesigned..."

Reply:
Maybe alter the DESCRIPTION a bit, this looks a bit better I think in HaikuDepot? ;)
![Project-Conceptor](https://user-images.githubusercontent.com/16057090/69863531-614fb300-129d-11ea-9cdb-c5a7da9b60e0.png)

Reply:
Suggestion:
```
DESCRIPTION="ProjectConceptor is a modular application designed to view, edit \
and process information which can be represented by a graph structure.
You can create nodes, name them and add attributes:
* Different nodes can be freely connected, all with full undo and redo support.
* Even recording of macros is supported.
* Two different views to edit the nodes or connections are available at the \
moment."
```